src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -60,13 +61,19 @@ public Future<Integer> loadData(TenantAttributes attributes, String tenantId, Ma
       .onFailure(throwable -> Future.failedFuture(throwable.getCause()));
   }
 
+  private void buildDataLoadingParameters(TenantAttributes tenantAttributes, TenantLoading tl) {
+    if (isNew(tenantAttributes, "13.5.0")) {
```

Comment:
we use 13.5.0 here because reasons-for-closure OR acquisition-methods introduced in 13.5.0?